Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -235,26 +216,36 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
     }
   }
 
-  // construct the seeding tools
-  // covariance tool, extracts covariances per spacepoint as required
-  auto extractGlobalQuantities =
-      [=](const SimSpacePoint& sp, float, float,
-          float) -> std::pair<Acts::Vector3, Acts::Vector2> {
-    Acts::Vector3 position{sp.x(), sp.y(), sp.z()};
-    Acts::Vector2 covariance{sp.varianceR(), sp.varianceZ()};
-    return std::make_pair(position, covariance);
-  };
+  auto start_preparation = std::chrono::high_resolution_clock::now();
+  // Config
+  Acts::SpacePointContainerConfig spConfig;
+  spConfig.useDetailedDoubleMeasurementInfo =
+      m_cfg.seedFinderConfig.useDetailedDoubleMeasurementInfo;
+  // Options
+  Acts::SpacePointContainerOptions spOptions;
+  spOptions.beamPos = {0., 0.};
+
+  // Prepare interface SpacePoint backend-ACTS
+  ActsExamples::SpacePointContainer container(spacePointPtrs);
+  // Prepare Acts API
+  Acts::SpacePointContainer<decltype(container), Acts::detail::RefHolder>
+      spContainer(spConfig, spOptions, container);
+
+  std::cout << "nsp: " << spContainer.size() << std::endl;
```

Comment:
```suggestion
```

Reply:
will be removed once the PR is finalised